appLegacy/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
Remove this and `consumer-rules.pro`

Reply:
Removed, I'm curious what these are for and why they are auto-added.